custom/layouts/partials/head.ejs

```diff
@@ -13,11 +13,12 @@
         href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.0.0/styles/default.min.css">
   
     <% if (locals.inlineCSS) { %>
-      <style type="text/css"><%- inlineCSS %></style>
+      <script><%- inlineCSS %></script>
```

Comment:
this doesn't seem right?

Reply:
Used MiniCssExtractPlugin to extract the two different SASS files.